Add mac entitlements to fix ledger integration + app icons

da-kami opened the pull request:
Fix from here: https://github.com/node-hid/node-hid/issues/321

Additionally use the default directory settings from electron builder to avoid confusion:
https://www.electron.build/configuration/configuration

da-kami committed:
Add mac entitlements to fix ledger integration + app icons

thomaseizinger commented on package.json:
```diff
@@ -76,10 +76,6 @@
     },
-    "directories": {
-      "buildResources": "resources",
-      "output": "release"
```
I think we are depending on this directory being named `release` in our GitHub workflows.

thomaseizinger replied:
You will need to adapt them :)

da-kami committed:
Generate icons with script to be able to change logo easily

Script only tested on Mac.
Script requires several tools to be installed.
Script not optimized :)

da-kami committed:
Change all relevant release occurrences to dist

da-kami committed:
Remove unused vscode config
